spec.bs

```diff
@@ -5473,21 +5484,30 @@ To <dfn>fetch the current outstanding trusted signals batch</dfn> given a
 {{unsigned short}}-or-null |experimentGroupId|, an [=origin=] |topLevelOrigin|,
 and a [=string=] |slotSizeQueryParam|:
 
+  1. If |signalsUrl| is null, return.
   1. Let |biddingSignalsUrl| be the result of [=building trusted bidding signals url=] with
      |signalsUrl|, |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/keys=],
      |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/ig names=],
      |experimentGroupId|, |topLevelOrigin|, and |slotSizeQueryParam|.
   1. Let « |partialTrustedBiddingSignals|, |partialPerInterestGroupData|, |dataVersion| » be the
      result of [=fetching trusted signals=] with |biddingSignalsUrl| and true.
-  1. [=map/For each=] |key| → |value| in |partialTrustedBiddingSignals|:
-    1. Set |trustedBiddingSignalsBatcher|'s
-      [=trusted bidding signals batcher/all trusted bidding signals=][|key|] to |value|.
-  1. [=set/For each=] |igName| item of |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/ig names=]:
-    1. [=map/Set=] |trustedBiddingSignalsBatcher|'s [=trusted bidding signals batcher/data versions=][|igName|] to |dataVersion|.
-  1. [=Append to a bidding signals per-interest group data map=] with
-      |partialPerInterestGroupData|, |trustedBiddingSignalsBatcher|'s
-      [=trusted bidding signals batcher/ig names=], and |trustedBiddingSignalsBatcher|'s
-      [=trusted bidding signals batcher/all per interest group data=].
+  1. If |partialTrustedBiddingSignals| is not null:
+    1. [=map/For each=] |key| → |value| in |partialTrustedBiddingSignals|:
+      1. Set |trustedBiddingSignalsBatcher|'s

```

Comment:
Just for readability, I'd probably bring this up a line at the same level as the for loop, since there is only a single step inside the for loop.

Reply:
Done